helm-charts/common/retriever-usvc/templates/deployment.yaml

```diff
@@ -82,6 +82,8 @@ spec:
           volumeMounts:
             - mountPath: /tmp
               name: tmp
+            - mountPath: /home/user/nltk_data
```

Comment:
Do we have this directory? Where it is created?

Reply:
nltk_data directory is created by the service itself during the runtime